include/LightGBM/utils/common.h

```diff
@@ -405,17 +405,18 @@ inline void SortForPair(std::vector<T1>& keys, std::vector<T2>& values, size_t s
 * cf. https://en.wikipedia.org/wiki/Gaussian_function
 *
 * y is a prediction.
-* t mesas true target.
+* t means true target.
+* g means gradient.
 * w means weights.
 */
-inline static double ApproximateHessianWithGaussian(double y, double t, double w=1.0f) {
+inline static double ApproximateHessianWithGaussian(const double y, const double t, const double g, const double w=1.0f) {
   const double diff = y - t;
   const double pi = M_PI;
```

Comment:
it seems need add ```#define _USE_MATH_DEFINES``` in windows:
http://stackoverflow.com/questions/6563810/m-pi-works-with-math-h-but-not-with-cmath-in-visual-studio